enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/integration/bolt/KerbNativeIT.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.server.security.enterprise.auth.integration.bolt;
+
+import org.junit.Test;
+
+import java.util.Map;
+import java.util.function.Consumer;
+
+import org.neo4j.graphdb.config.Setting;
+import org.neo4j.server.security.enterprise.configuration.SecuritySettings;
+
+public class KerbNativeIT extends EnterpriseAuthenticationTestBase
```

Comment:
Is this class really part of this bug-fix? Seems to be something for kerberos support instead.